Quotient/connection.cpp

```diff
@@ -637,7 +638,26 @@ void Connection::Private::consumeAccountData(Events&& accountDataEvents)
 
 void Connection::Private::consumePresenceData(Events&& presenceData)
 {
-    // To be implemented
+    for (const auto &event : presenceData) {
+        auto presenceEvent = eventCast<const PresenceEvent>(event);
+        auto sender = presenceEvent->fullJson()[u"sender"_s].toString(); // TODO ?
```

Comment:
```suggestion
        auto sender = presenceEvent->senderId();
```
Calling it `senderId()` to align with `RoomEvent`.